regression/ansi-c/function_return1/test.desc

```diff
@@ -1,7 +1,7 @@
 CORE
 main.c
 --verbosity 2
-^main.c:3:1: warning: function has return void but a return statement returning signed int$
+^.*: function has return void but a return statement returning signed int$
```

Comment:
Nit pick: just remove `^.*`
And just curious: what _does_ the output of goto-cl look like?

Reply:
file main.c line 3 function fun: function has return void but a return statement returning signed int

Reply:
We could eventually imitate the style of the real CL, which is
main.c(3): warning C4098: 'fun': 'void' function returning a value